### ISSUE TITLE
Synth Parameters

### ISSUE THREAD

**Issue by lennart** [tidalcycles/tidalcycles.github.io#2]
Currently all **Synth Parameters** are listed within the Patterns chapter. This however does not really make sense.

One idea would be to move them to the reference.

Is it feasible to make this its own subsection, and let each synth parameter have its own markdown file for documentation?

**Comment by kindohm** [comment]
I think this makes sense. There are enough synth params to fill up a new section. Plus, I imagine new Tidal users without a background in sound processing or music might not know what a lot of these parameters mean, so it makes sense to give each one a dedicated file.

**Comment by lennart** [comment]
I added a separate file for each parameter, additionally I fixed the sorting for menu/content so it's first sorted by an explicitly given weight (negative will go up, positive down) and then by title ascending, this way we have alphabetical sorting (case sensitive though).

To visualize things like the x-pass filter cutoff/resonance/q I thought about using webaudio API to add some interactivity to these snippets. However, the BiQuad filter of WebAudio API do not have a resonance value, but just a Q factor, which seems to be not identical (sorry, I cannot explain it better, just what I read within the source of one of the [WebAudio examples](http://www.html5rocks.com/en/tutorials/webaudio/intro/#toc-filter)).

Additionally it is possible to visualize the frequency response graph, which would be super nice in realtime, to make it even clearer what the given parameter will do. Given the possibilities of modern browser, we should try to make it as easy as possible to explain what is actually happening to the sound (for people coming from a non-musical background).

On the other hand, we should also explain a bit more about the possibilities and workings of haskell, or a functional programming language in general, for those coming from a non-programmer background. However, I am unsure whether this is something to be put into the soon to come "Hacking" section. This can be kept on a level that I found really nice when learning about "monads", "you don't need to understand how Monads work, but it helps to understand how to use them". In our case, the word **Monad** should **not** appear in the tidal docs… ;)
